src/main/java/com/uid2/shared/cloud/CloudStorageS3.java

```diff
@@ -51,6 +51,7 @@ public void upload(String localPath, String cloudPath) throws CloudStorageExcept
         try {
             File file = new File(localPath);
             this.s3.putObject(bucket, cloudPath, file);
+            this.checkVersioningEnabled();
```

Comment:
Can we instead check the value returned by `PutObjectResult.getVersionId()`?

https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/s3/model/PutObjectResult.html#getVersionId--